README.md

```diff
@@ -179,4 +170,4 @@ Drag and drop multiple files (Chrome and Firefox) or a folder (Chrome):
 
 That's all for now,
 
-Rene K. Mueller, Jeff Gay, Mark Moissette and JSCAD Organization
```

Comment:
No more spotlight, fame & fortune for us ? :)

Reply:
Well... Rene doesn’t really take part... and I’m shy. ☺️ 

I’d prefer to have sponsors and contributors. 😃